resource/ignite2019-prod/session_map.json

```diff
@@ -155,6 +155,28 @@
         "My Schedule",
         "Schedule Builder - Remote"
       ]
+    },
+    {
+      "SessionTypeName": "OpenHack",
+      "HubbId": 3317,
+      "SessionSets": [
+        "Session Scheduler",
```

Comment:
Needs to be part of a separate Session Set and NOT part of the Session Scheduler one, 
This will allow us to restrict visibility of that session type to a sub set of attendees.

Reply:
We also need to add a new entry into the rules.json file to map the OpenHackAttendee type see the new session set for the OpenHack session type